src/gh_pr/utils/config.py

```
@@ -8,6 +8,17 @@
 except ImportError:
     import tomli as tomllib
 
+# Constants for test compatibility
+ALLOWED_CONFIG_DIRS = ['/home', '/tmp', '/etc']
```

Ruff (0.13.1)

Probable insecure usage of temporary file or directory: "/tmp"

(S108)